common/src/hand_off.rs

```diff
@@ -31,14 +31,14 @@ pub struct FirmwareHandoffTable {
     pub fips_fw_load_addr: u32,
 
     /// Physical base address of Runtime FW Module in ICCM SRAM.
-    pub rt_fw_load_addr: u32,
-
-    /// Entry point of Runtime FW Module in ICCM SRAM.
-    pub rt_fw_entry_point: u32,
+    pub rt_fw_load_addr: u8,
 
     /// Index of FMC TCI value in the Data Vault.
     pub fmc_tci_dv_idx: u8,
 
+    /// Entry point of Runtime FW Module in ICCM SRAM.
```

Comment:
The order here is odd. Either move this back to where the other address are, or group with all the other RT fields. Either way, make this and the doc match.

If you did this for alignment, we can also add another reserved field after the group of address _idx fields. If you make the FIPS address field the last of those, we would have some room if we find that we need more data once that feature is supported.

Reply:
This is probably rust-analyzer changing field order when I saved.